source/release-notes/1.8.txt

```diff
@@ -21,9 +21,91 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
 
-New Features:
+Embedded Verifier
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.8, ``mongosync`` includes an optional embedded
+verifier that performs a series of checks against the source and 
+destination clusters to verify a successful sync.
+
+For more information, see :ref:`c2c-embedded-verifier`.
+
+Other Notes
+~~~~~~~~~~~
+
+Updates to ``mongosync`` ``/progress`` API endpoint:
+
+- The ``/progress`` endpoint now returns a top-level boolean 
+  ``success`` field.
+
+- The ``/progress`` endpoint now returns the integer 
+  ``totalEventsApplied`` field, which approximates the number of 
+  change events applied to a destination cluster by an instance of 
+  ``mongosync``.
+
+- The ``/progress`` endpoint now reports per-partition progress 
+  tracking with the following changes:
+
+  - Calling ``/progress`` before ``mongosync`` reaches the collection 
+    copy phase will return 0 for ``estimatedCopiedBytes`` and 1 for 
+    ``estimatedTotalBytes``.
+
+  - ``estimatedTotalBytes`` may change throughout the collection copy 
+    phase if documents are inserted or deleted on the source cluster.
+
+  - ``estimatedCopiedBytes`` will never be greater than 
+    ``estimatedTotalBytes``. Progress will reach 100% at the end of 
+    the collection copy phase 
+    (``estimatedCopiedBytes`` = ``estimatedTotalBytes``).
+
+  - When performing a live upgrade from an earlier version to 1.8.0 or 
+    higher, the collection copy data will start over from 
+    ``[0 bytes copied / 1 bytes total]``. After a live upgrade, 
+    ``/progress`` only reports the progress of data copied after the 
+    upgrade completed.
+
+Optimizations:
+
+- Significantly reduced the time needed to create partitions and added logging 
+  that tracks the partition creation process.
+
+Other Changes:
+
+- ``mongosync`` now performs a round robin assignment of the primary 
+  shard for each database if the destination is a sharded cluster.
+
+- ``mongosync`` now errors if it cannot create an index after 10 
+  attempts. Previously, the system would attempt to create an index 
+  indefinitely.
+
+- ``mongosync`` now sends hostnames as telemetry.
+
+- Improved performance of the initialization process by eliminating 
+  the creation of unnecessary dummy indexes. ``mongosync`` now creates 
+  dummy indexes for sharded collections only if the destination 
```

Comment:
(wording)
Can we use a word other than "dummy"?

Reply:
I had the same thought as well, but the [docs](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/api/start/#:~:text=mongosync%20builds%20dummy%20indexes%20that,retains%20this%20index%20after%20commit.) [currently](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/disaster-recovery/#:~:text=Dummy%20Indexes,word%20dummy.) use the term `dummy index` and I thought it was best to stay consistent.

I'm open to changing this, but wanted to share that extra context. Let me know what you think.

Reply:
Makes sense. Thanks for talking it through.